org.tahoma2d.Tahoma2D.yaml

```diff
@@ -12,7 +8,7 @@ finish-args:
   - --socket=x11
   - --device=dri
   - --socket=pulseaudio
-  - --filesystem=home
+  - --filesystem=host
```

Comment:
why broadening this?

Reply:
I believe this was copied from OT manifest.

Reply:
do wrongs don't make a right. The permission is very broad and it's hard to narrow it without breaking it for the users.

Reply:
If OT chose this setting (and apparently was allowed), there must be a reason for it.  If so, my users would also need the same level of access.

Some people have both OT and T2D installed and can transfer projects between them.  This would not be good if T2D user sudden;y can't use something created in OT because of access restrictions.

I'm requesting this be maintained.  If OT is asked to change and changes it, I will also change.  For interoperability between OT and T2D I try to keep things as consistent as possible.